planning/standards-status.md

```diff
@@ -0,0 +1,83 @@
+---
+description: This list collects standards that the Thunderbird family of applications currently at least partially implements or supports (in our code base, ignoring things like TCP which we inherit from upstream components). It reflects the state of the current latest main repo code.
+---
+
+## Mail
+
+| Spec Name                                                          | Desktop Status | Android Status | Standards Involvement      | Comment |
+|:------------------------------------------------------------------ |:--------------:|:--------------:|:-------------------------- |:------- |
+| [POP3](https://www.rfc-editor.org/rfc/rfc1939)                     |  &#10004;      |   &#10004;     |                            |         |
+| [IMAP 4r1](https://datatracker.ietf.org/doc/html/rfc3501)          |  &#10004;      |   &#10004;     |                            |         |
+| [IMAP IDLE](https://datatracker.ietf.org/doc/html/rfc2177)         |  &#10004;      |   &#10004;     |                            |         |
+| [SMTP](https://www.rfc-editor.org/rfc/rfc821)                      |  &#10004;      |   &#10004;     |                            |         |
+| [NNTP](https://datatracker.ietf.org/doc/html/rfc977)               |  &#10004;      |   &#10004;     |                            |         |
+
+## RSS Feeder
+
+| Spec Name                                                          | Desktop Status | Android Status | Standards Involvement      | Comment |
+|:------------------------------------------------------------------ |:--------------:|:--------------:|:-------------------------- |:------- |
+| [RSS](https://www.rssboard.org/rss-specification)                  |  &#10004;      |      N/A       |                            | RSS 0.9 vs 1.0 vs 2.0 are rather different. Which do we support?  |
+| [Atom](https://www.rfc-editor.org/rfc/rfc4287)                     |  &#10004;      |      N/A       |                            |         |
+
+## Security and Authentication
+
+| Spec Name                                                          | Desktop Status | Android Status | Standards Involvement      | Comment |
+|:------------------------------------------------------------------ |:--------------:|:--------------:|:-------------------------- |:------- |
+| [MIME](https://www.rfc-editor.org/rfc/rfc2045)                     |  &#10004;      |   &#10004;     |                            | MIME is actually over 3 RFCs: [RFC2045](https://www.rfc-editor.org/rfc/rfc2045), [RFC2046](https://www.rfc-editor.org/rfc/rfc2046), and [RFC2047](https://www.rfc-editor.org/rfc/rfc2047). All 3 are supported for both Desktop and Android platforms. |
+| [List Management Headers](https://www.rfc-editor.org/rfc/rfc2369)  |                |                |                            | See [Bug 29041](https://bugzilla.mozilla.org/show_bug.cgi?id=29041)  |
```

Comment:
I think we actually support at least some of this on desktop now...

Reply:
Yeah, @mkmelin added support in https://bugzilla.mozilla.org/show_bug.cgi?id=29041. And https://bugzilla.mozilla.org/show_bug.cgi?id=512385 added RFC 5064.

Reply:
Ok I've updated the List Management Headers line to reflect the desktop support as well as the 2 RFCs.